### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by NuhuWambali** [community/community#67493] [New to GitHub] [answered]
i got this message whenever i tried to push my projects to github! anyone knows how to solve...

**Comment by NuhuWambali** [comment, marked as answer]
ive contacted github support team and they help me to solve this issue

**Reply by Amdadullah-Raihan** [reply]
How did you contact the GitHub support team? Is it possible for free users?

**Comment by mustafaalobaidy** [comment]
Hello

**Reply by mustafaalobaidy** [reply]
You are marked as spam, and therefore cannot authorize a third party application.

**Reply by rababR1** [reply]
Same issue here. I get: This account is flagged, and therefore cannot authorize a third party application.

**Comment by usamanadeemdeveloper** [comment]
> I got this message whenever I tried to push my projects to github! anyone knows how to solve...

Please help me I can't log into any account via Github even I'm not able to push or pull code via Vs code!

**Reply by fatihdeli993** [reply]
Türkçe proğlama

**Reply by rababR1** [reply]
> > I got this message whenever I tried to push my projects to github! anyone knows how to solve...
> 
> Please help me I can't log into any account via Github even I'm not able to push or pull code via Vs code!

Exact same thing happening to me!

**Reply by rababR1** [reply]
BTW, where do I contact GitHub support?
